unity/EditorPlugin/EditorPluginNet46.csproj

```diff
@@ -40,8 +41,9 @@
     <Compile Remove="AssetPostprocessors\CsprojAssetPostprocessor.cs" />
     <Compile Remove="AssetPostprocessors\SlnAssetPostprocessor.cs" />
     <Compile Remove="RiderMenu.cs" />
+    <Compile Remove="PluginSettingsPrePackage.cs" />
```

Comment:
What's going on with this file? Why is it split out, and if it's pre-package, why is it included in a project for Unity 2019.2?

Reply:
[PreferenceItem("Rider")] is not getting called with Unity 2019.2+, but it causes compiler warning. So I just tried to avoid that compiler warning.